torchgen/model.py

```diff
@@ -187,13 +187,19 @@ class DispatchKey(Enum):
     # END autogenerated
 
     def __str__(self) -> str:
+        return self.name.replace("PrivateUse1", PrivateUse1Name.custom_name)
+
+    # for PrivateUse1, it needs to have two representations of str.Most of the time, it should be the name which user
+    # set, but when the operator is actually registered, it needs to be made PrivateUse1.
```

Comment:
Are you sure? It seems to me like most of the time you want PrivateUse1 when this dispatch key is printed throughout torchgen

Reply:
I'm sure, because of our custom device has its own name(e.g. YPU), it's better to be 'YPU' in this code generated with torchgen. But, when I register our operator to my custom device, its str behavior must be "PrivateUse1" rather than "YPU".